source/includes/environment_config/_30-create.md.erb

```diff
@@ -0,0 +1,119 @@
+## Create an environment
+
+```shell
+$ curl 'https://ci.example.com/go/api/admin/environments/' \
+      -u 'username:password' \
+      -H 'Accept: application/vnd.go.cd.v1+json' \
+      -H 'Content-Type: application/json' \
+      -X POST -d '{
+        "name" : "new_environment",
+        "pipelines" : [
+          {
+          "name" : "up2"
+          }
+        ],
+        "agents" : [
+          {
+            "uuid" : "12345678-e2f6-4c78-123456789012"
+          }
+        ],
+        "environment_variables" : [
+          {
+            "name" : "username",
+            "value" : "admin",
+            "secure" : false
+          }
+        ]
+      }'
+```
+
+> The above command returns JSON structured like this:
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/vnd.go.cd.v1+json; charset=utf-8
+```
+
+```json
+{
+  "_links": {
+    "self": {
+      "href": "https://ci.example.com/go/api/admin/environments/new_environment"
+    },
+    "doc": {
+      "href": "http://api.go.cd/#environment_config"
```

Comment:
no underscores in url fragments. None of the existing urls have underscores in them. Please fix the representer and this